Выполненный проект 4 спринта

Yuuki-that-one opened the pull request.

Commit:
Выполненный проект 4 спринта

So1ovyov commented on MovieQuiz/AppDelegate.swift:
```diff
@@ -5,8 +5,8 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
-    ) -> Bool {
-        print("Hello world for branch to merge")
+    ) -> Bool {    // Override point for customization after application launch.
```
Этот комментарий можно удалить

So1ovyov commented on MovieQuiz/Helpers/UIColor+Extensions.swift:
```diff
@@ -1,3 +1,10 @@
+        static var ypBackground: UIColor { UIColor(named: "YP Background") ?? UIColor.darkGray }
+        static var ypGray: UIColor { UIColor(named: "YP Gray") ?? UIColor.gray }
+        static var ypWhite: UIColor { UIColor(named: "YP White") ?? UIColor.white}
+     }
```
Кажется сместился отступ у закрывающей фигурной скобки

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -4,69 +4,180 @@ final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
```
В этом классе нарушен порядок элементов, его стоит соблюдать для удобочитаемости кода
https://habr.com/ru/articles/744470/

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -4,69 +4,180 @@ final class MovieQuizViewController: UIViewController {
+    }
+    
+    // вью модель для состояния "Вопрос показан"
+    struct QuizStepViewModel {
```
Для этой структуры и двух следующих можно добавить модификатор private

So1ovyov commented on MovieQuiz/Presentation/MovieQuizViewController.swift:
```diff
@@ -4,69 +4,180 @@ final class MovieQuizViewController: UIViewController {
+    private func showAnswerResult(isCorrect: Bool) {
+        if isCorrect {
+                correctAnswers += 1
+            }
```
Сместился отступ

Commit:
Исправления проекта по замечаниям
Удален лишний коммент
Исправлена верстка и порядок расположения элементов в классе
Добавлены скругленные углы для постеров
Рамка пропадает после загрузки следующего вопроса
После нажатия кнопки не реагируют до показа следующего вопроса
